daprdocs/content/en/contributing/contributing-overview.md

```diff
@@ -54,8 +54,48 @@ All contributions come through pull requests. To submit a proposed change, follo
     - Code changes require tests
 1. Update relevant documentation for the change
 1. Commit and open a PR
-1. Wait for the CI process to finish and make sure all checks are green
-1. A maintainer of the project will be assigned, and you can expect a review within a few days
+
+    - Developer Certificate of Origin: Signing your work
+    
+        The Developer Certificate of Origin (DCO) is a lightweight way for contributors to certify that they wrote or otherwise have the right to submit the code they are contributing to the project. Here is the full [text of the DCO](https://developercertificate.org/), reformatted for readability:
```

Comment:
nitpick:
```suggestion
        The Developer Certificate of Origin (DCO) is a lightweight way for contributors to certify that they wrote or otherwise have the right to submit the code they are contributing to the project. Here is the full text of the [DCO](https://developercertificate.org/), reformatted for readability:
```